src/main/webapp/js/webrtc_adaptor.js

```diff
@@ -256,7 +256,9 @@ function WebRTCAdaptor(initialValues)
 			});
 		}
 		else {
-			stream.addTrack(audioStream.getAudioTracks()[0]);
+			if(mediaConstraints.audio != false){
```

Comment:
I'm not sure about the error but I think checking `audiostream.getAudioTracks` is a better idea. Because it's the root cause of the problem. `mediaConstraints.audio` is like a symptom and we may change `mediaConstraints.audio` to some other value than false and we may start to get this or similar problem again.

Reply:
Thanks for the great feedback! I added check `audioStream` is `undefined` case in a new commit. If we disabled audio stream, `audioStream` parameter returning `null`. So, we need to check  `audioStream` is `undefined`.